track_odometry/src/track_odometry.cpp

```diff
@@ -298,6 +298,11 @@ class TrackOdometryNode
   {
     neonavigation_common::compat::checkCompatMode();
 
+    bool use_tcp_no_delay;
+    pnh_.param("use_tcp_no_delay", use_tcp_no_delay, false);
```

Comment:
I think the default value is OK to be true.

Reply:
Done